server/asset/eth/eth.go

```diff
@@ -30,6 +33,7 @@ const (
 	// The blockPollInterval is the delay between calls to bestBlockHash to
 	// check for new blocks.
 	blockPollInterval = time.Second
+	requiredNPeers    = 2
```

Comment:
Any reason to have 2 instead of 1? I'm wondering if this'll still work when we expand the harness tests.

Reply:
Same thought.  It's common in production to have a single peer where that peer is one you control, like a proxy node.

Reply:
For better chances of reporting isSyncing correctly. Yeah, the harness would require a third node with this I suppose...

Reply:
no longer looking at peers